tox.ini

```diff
@@ -1120,20 +1091,6 @@ commands =
 
   test-util-http: pytest {toxinidir}/util/opentelemetry-util-http/tests {posargs}
 
-  test-sdk-extension-aws: pytest {toxinidir}/sdk-extension/opentelemetry-sdk-extension-aws/tests {posargs}
```

Comment:
The size of this `tox.ini` file will be greatly reduced when the rest of the packages are refactored.